homework_9_071222/src/myMathMethods/MyMath.java

```diff
@@ -0,0 +1,105 @@
+// 2. Создать собственную реализацию класса Math со статическими методами сложения,
+// вычитания, умножения, деления, возвращения большего числа,
+// меньшего числа(на вход два параметра), находнения большего целого числа,
+// меньшего целого числа - округление(на вход число с плавающей точкой),
+// возведение в степень, нахождение процента, квадратный корень
+
+package myMathMethods;
+
+public class MyMath {
+
+    public static int sum(int a, int b) {
+        return a + b;
+    }
+
+    public static double sum(double x, double y) {
+        return x + y;
+    }
+
+    public static int subtraction(int a, int b) {
+        return a - b;
+    }
+
+    public static double subtraction(double x, double y) {
+        return x - y;
+    }
+
+    public static int multiplication(int a, int b) {
+        return a * b;
+    }
+
+    public static double multiplication(double x, double y) {
+        return x * y;
+    }
+
+    public static int division(int a, int b) {
+        return a / b;
+    }
+
+    public static double division(double x, double y) {
+        return x / y;
+    }
+
+    //    возвращения большего числа, меньшего числа(на вход два параметра)
+    public static int maxValue(int a, int b) {
+        if (a >= b) {
+            return a;
+        } else {
+            return b;
+        }
+    }
+
+    public static int minValue(int a, int b) {
+        if (a >= b) {
+            return b;
+        } else {
+            return a;
+        }
+    }
+
+    //    находнения большего целого числа, меньшего целого числа - округление(на вход число с плавающей точкой)
+    public static double maxValue(double x, double y) {
+        x = Math.round(x);
```

Comment:
what if you get x = 2.6, y=2.5?

Reply:
Fixed. Numbers are compared first, then rounded up.